data/core/interactive.md

```diff
@@ -696,37 +696,6 @@
 * affirm
     - utter_thumbsup
 
-## Story from conversation with 03727bb9d4cb4f64afdc383818739680 on January 17th 2019
```

Comment:
why delete this?

Reply:
I'll redo all the interactive conversations related to the getting started flow as there are too many conflicts with the old ones.